src/zui/ZUICreatePerson/PersonalInfoForm.tsx

```diff
@@ -0,0 +1,278 @@
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { ExpandMore } from '@mui/icons-material';
+import utc from 'dayjs/plugin/utc';
+import {
+  Box,
+  Button,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+} from '@mui/material';
+import dayjs, { Dayjs } from 'dayjs';
+import { FC, useEffect, useRef } from 'react';
+
+import globalMessageIds from 'core/i18n/globalMessageIds';
+import { makeNaiveDateString } from 'utils/dateUtils';
+import messageIds from 'zui/l10n/messageIds';
+import { TagManagerSection } from 'features/tags/components/TagManager';
+import useCustomFields from 'features/profile/hooks/useCustomFields';
+import { useNumericRouteParams } from 'core/hooks';
+import useTags from 'features/tags/hooks/useTags';
+import { checkInvalidFields, ShowAllTriggeredType } from '.';
+import { Msg, useMessages } from 'core/i18n';
+import { ZetkinCreatePerson, ZetkinTag } from 'utils/types/zetkin';
+
+dayjs.extend(utc);
+
+type GenderKeyType = 'f' | 'm' | 'o' | 'unknown';
+type ZetkinCreatePersonFields = keyof Omit<
+  ZetkinCreatePerson,
+  'tags' | 'customFields'
+>;
+
+interface PersonalInfoFormProps {
+  debounced: (
+    field: keyof ZetkinCreatePerson,
+    value: string | null,
+    custom: boolean
+  ) => void;
```

Comment:
What is the purpose of this? This is an indication of poor separation of concerns IMO. The debouncing should not be the concern of two components. It should be either in this one, or in the outside one, not "in both" (as becomes the case when the interface between the two components contains something referenced as `debounced`).